app/src/main/res/layout/dialog_confirmation.xml

```diff
@@ -0,0 +1,89 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:tools="http://schemas.android.com/tools"
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    >
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/scrim"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:paddingStart="30sp"
+        android:paddingEnd="30sp"
+        android:paddingBottom="50sp"
```

Comment:
@danacealfonso 
Use correct unit(dp) for padding dimensions.

Reply:
Change unit in a6bca1a675a3acbf93e208e450a0966b79bea1c5